src/modules/client/components/planning/DeleteEventsModal.vue

```diff
@@ -64,8 +72,20 @@ export default {
         { label: 'A partir d\'une date', value: false },
       ],
       loading: false,
+      isCustomerAbsence: true,
+      customerAbsenceOptions: [
+        { label: 'Congés', value: LEAVE },
+        { label: 'Hospitalisation', value: HOSPITALIZATION },
+        { label: 'Autre', value: OTHER },
+      ],
     };
   },
+  watch: {
+    'deletedEvents.inRange': function () {
+      if (!this.deletedEvents.inRange) this.isCustomerAbsence = false;
```

Comment:
Que penses-tu de mettre `this.isCustomerAbsence = this.deletedEvents.inRange` ? Ca permettrait de conserver le fait que par defaut c'est une absence beneficiaire